src/logic.js

```diff
@@ -206,6 +206,7 @@ define([
                 if (!refMug &&
                     (!mug.options.ignoreReferenceWarning || !mug.options.ignoreReferenceWarning(mug)) &&
                     _this.opts.allowedDataNodeReferences.indexOf(pathWithoutRoot) === -1 &&
+                    !pathWithoutRoot.startsWith('meta/') &&
```

Comment:
Worth updating https://github.com/dimagi/Vellum/blob/master/tests/options.js#L93-L98 ?  Is allowedDataNodeReferences now ever used for paths other than `/data/meta/*`?